src/components/menu/style.css

```diff
@@ -0,0 +1,67 @@
+.menuToggle {
+  opacity: 0;
+}
+.menu__btn {
+  position: fixed;
+  top: 20px;
+  left: 20px;
+  width: 26px;
+  height: 26px;
+  cursor: pointer;
+  z-index: 1;
+  background-color: unset;
+  border: none;
+}
+.menu__btn > span,
```

Comment:
It would be better if you give this span a class name instead of refering it like this.